lib/pages/Profile.dart

```diff
@@ -43,145 +44,150 @@ class Profile extends StatelessWidget {
+                                      style: TextStyle(
+                                        fontSize: 22,
+                                        fontWeight: FontWeight.bold,
+                                      )
+                                  ),
+                                  SizedBox(height: 4),
+                                  Semantics(
+                                    container: true,
+                                    child: Text('Joined ' + riderProvider.info.joinDate,
+                                        style: TextStyle(
+                                          fontSize: 14,
+                                          color: Theme.of(context).accentColor,
+                                        )),
+                                  )
+                                ],
+                              ),
+                            )
+                          ]))),
+                  sectionDivider,
+                  InfoGroup(
+                    'Account Info',
+                    [
+                      InfoRow(
+                          Icons.mail_outline,
+                          riderProvider.info.email,
+                          'Email'
                       ),
-                    )
-                  ]))),
-              sectionDivider,
-              InfoGroup(
-                'Account Info',
-                [
-                  InfoRow(
-                    Icons.mail_outline,
-                    riderProvider.info.email,
-                  ),
-                  InfoRow(Icons.phone, riderProvider.info.phoneNumber,
-                      editPage:
-                          EditPhoneNumber(riderProvider.info.phoneNumber)),
-                  InfoRow(
-                    Icons.person,
-                    riderProvider.info.firstName +
-                        ' ' +
-                        riderProvider.info.lastName,
-                    editPage: EditName(riderProvider.info.firstName,
-                        riderProvider.info.lastName),
+                      InfoRow(
+                        Icons.phone,
+                        riderProvider.info.phoneNumber,
+                        'Phone number',
+                        editPage: EditPhoneNumber(riderProvider.info.phoneNumber),
+                        readDigits: true,
+                      ),
+                      InfoRow(
+                        Icons.person,
+                        riderProvider.info.firstName +
+                            ' ' +
+                            riderProvider.info.lastName,
+                        'Name',
+                        editPage: EditName(riderProvider.info.firstName,
+                            riderProvider.info.lastName),
+                      ),
+                    ],
                   ),
+                  sectionDivider,
+                  TermsInfo(),
+                  sectionDivider,
+                  SignOutButton(),
```

Comment:
When it's the first time using the app/login, this button goes to the home screen when pressed. If my fixes with navigation already addresses this, then this is not an issue.

Reply:
Fixed this up, thanks!